src/MyPlot/provider/SQLiteDataProvider.php

```diff
@@ -20,7 +20,7 @@ class SQLiteDataProvider extends DataProvider
     public function __construct(MyPlot $plugin, $cacheSize = 0) {
         parent::__construct($plugin, $cacheSize);
 
-        $this->db = new \SQLite3($this->plugin->getDataFolder()."Data\\plots.db");
+        $this->db = new \SQLite3($this->plugin->getDataFolder()."/Data/plots.db");
```

Comment:
In the getDataFolder function, there is already a directory separator given at the and